addons/project/models/res_config_settings.py

```diff
@@ -9,7 +9,6 @@ class ResConfigSettings(models.TransientModel):
 
     module_project_forecast = fields.Boolean(string="Planning")
     module_hr_timesheet = fields.Boolean(string="Task Logs")
-    group_subtask_project = fields.Boolean("Sub-tasks", implied_group="project.group_subtask_project")
```

Comment:
Requires a migration script. 

Note: You don't have to start the migration script now, it's just for your information. ;)